src/client/components/Prices.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+class Prices extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      prices: []
+    }
+  }
+  componentDidMount() {
+      //czego tutaj nie dajemy localhost?
+    fetch('/api/prices', {
```

Comment:
nie dajemy hosta, bo jak bedziemy chcieli potem odpalić na innej domenie (docelowo) to musielibysmy budowac oddzielny bundle dla kazdego hosta